### ISSUE TITLE
[bug]: Search auto populate results not accessible with tab or up/down keys.

### ISSUE THREAD

**Issue by RanjitBShinde** [magento/pwa-studio#4022]
The keyboard focus and tab order does not work as expected. When we try to access auto populated search results with tab or key up/down key, focus is not going to search results. Instead of search results focus is going to the main page.

Step to reproduce : 
1. Goto https://venia.magento.com/
2. Click on search icon on home page
3. Search any keyword for example, "Night"
4. Once search result appears try to access search results by tab or up/down keys

Actual Result : 
User is not able to access auto populated search results with tab or up/down keys

Expected Result : 
User should able to access auto populated results with the tab or key/up down keys.

<img width="1052" alt="image" src="https://user-images.githubusercontent.com/16210367/209907519-16b99996-eda3-4798-a74c-6f7b71f8c183.png">

**Comment by anthoula** [comment]
@adobe export issue to JIRA project PWA as Bug

**Comment by Sahilsks** [comment]
@magento I am working on this

**Comment by glo82145** [comment]
The ticket associated with this issue has been merged hence closing this issue
